std/assembly/math.ts

```diff
@@ -1230,6 +1232,20 @@ export namespace NativeMath {
   }
 }
 
+/** @internal */
+@lazy var rempio2f_y: f64;
+@lazy const PIO2_TABLE: u64[] = [
+  0xA2F9836E4E441529,
+  0xFC2757D1F534DDC0,
+  0xDB6295993C439041,
+  0xFE5163ABDEBBC561
+];
+
+/** @public Used as return values from Mathf.sincos */
+@global @lazy var sincos_s32: f32;
+@global @lazy var sincos_c32: f32;
```

Comment:
What do you think of making these part of the Mathf namespace, next to `sincos` itself?

Reply:
Hmm, yeah. Makes sense